src/auth/controllers/auth_login.js

```diff
@@ -0,0 +1,37 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const User = require('../../users/user_schema');
+const { secret } = require('../../config');
+
+const passwordMatches = (password1, hash) => bcrypt.compareSync(password1, hash);
+
+const generateToken = paramsForTokenGeneration => {
+  return jwt.sign(paramsForTokenGeneration, secret, {
+    expiresIn: 60 * 60 * 24
```

Comment:
move this magic number to config 

https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)